src/pages/workspace/card/issueNew/ConfirmationStep.tsx

```diff
@@ -44,18 +74,53 @@ function ConfirmationStep() {
                     stepNames={CONST.EXPENSIFY_CARD.STEP_NAMES}
                 />
             </View>
-            <Text style={[styles.textHeadlineLineHeightXXL, styles.ph5, styles.mv3]}>{translate('workspace.card.issueNewCard.letsDoubleCheck')}</Text>
-            <View style={[styles.ph5, styles.mb5, styles.flex1, styles.justifyContentEnd]}>
-                {/* TODO: the content will be created in https://github.com/Expensify/App/issues/44309 */}
-                <Button
-                    isDisabled={isOffline}
-                    success
-                    large
-                    style={[styles.w100]}
-                    onPress={submit}
-                    text={translate('workspace.card.issueCard')}
+            <ScrollView
+                style={styles.pt0}
+                contentContainerStyle={styles.flexGrow1}
+            >
+                <Text style={[styles.textHeadlineLineHeightXXL, styles.ph5, styles.mv3]}>{translate('workspace.card.issueNewCard.letsDoubleCheck')}</Text>
+                <Text style={[styles.textLabelSupporting, styles.ph5, styles.mv3]}>{translate('workspace.card.issueNewCard.willBeReady')}</Text>
+                <MenuItemWithTopDescription
+                    description={translate('workspace.card.issueNewCard.cardholder')}
+                    title={PersonalDetailsUtils.getPersonalDetailByEmail(data?.assigneeEmail ?? '')?.displayName}
+                    shouldShowRightIcon
+                    onPress={() => editStep(CONST.EXPENSIFY_CARD.STEP.ASSIGNEE)}
                 />
-            </View>
+                <MenuItemWithTopDescription
+                    description={translate('workspace.card.issueNewCard.cardType')}
+                    title={data?.cardType ? translate(`workspace.card.issueNewCard.${data?.cardType}Card`) : ''}
```

Comment:
IMO, Adding a condition or using a switch case will be more Comprehensible. Anyway, the current code also work well